.github/workflows/windows.yml

```diff
@@ -128,7 +128,10 @@ jobs:
           version: "v0.7.5"
 
       - name: Install build dependencies
-        run: choco install --no-progress ninja
+        run: |
+          # Source script with `choco` wrapper that provides the retrying mechanism
+          . ${{ env.OPENVINO_REPO }}/.github/utils/ChocoHelper.ps1
+          Choco-Install -PackageName ninja
```

Comment:
can we download ninja from github https://github.com/ninja-build/ninja/releases ?

Reply:
We could but I think we would need to include it into `PATH` and/or other Win-specific places in the steps where it would be used as `PATH` sets are not persisted between steps iirc.

Reply:
Alright, I try it in a separate PR and then we will decide which one would look/work best.

Reply:
So, here is the [PR](https://github.com/openvinotoolkit/openvino/pull/22570) with the direct from-GitHub downloading. If the newer one looks better, feel free to approve it and I'll close this one.

Reply:
I prefer that one